gitlabform/processors/group/group_saml_links_processor.py

```diff
@@ -0,0 +1,55 @@
+from logging import debug
+from typing import List
+
+from gitlabform.gitlab import GitLab
+from gitlab.v4.objects import Group
+from gitlabform.processors.abstract_processor import AbstractProcessor
+
+
+class GroupSAMLLinksProcessor(AbstractProcessor):
+
+    def __init__(self, gitlab: GitLab):
+        super().__init__("saml_group_links", gitlab)
+
+    def _process_configuration(self, group_path: str, configuration: dict) -> None:
+        """Process the SAML links configuration for a group."""
+
+        configured_links = configuration.get("saml_group_links", {})
+        enforce_links = configuration.get("saml_group_links|enforce", False)
+
+        group: Group = self.gl.get_group_by_path_cached(group_path)
+        existing_links: List[dict] = self._fetch_saml_links(group)
```

Comment:
can you return a List of Link objects rather than a List of dict? I think that'll make code cleaner